sources/sections-en/02-system-of-units.adoc

```diff
@@ -356,13 +356,13 @@ Physical quantities can be organized in a system of dimensions, where the system
 |===
 | Base quantity | Typical symbol for quantity | Symbol for dimension
 
-| time | stem:[t] | stem:[sf "T"]
-| ((length)) | stem:[l, x, r], etc. | stem:[sf "L"]
-| ((mass)) | stem:[m] | stem:[sf "M"]
-| ((electric current)) | stem:[ii(I), i] | stem:[sf "I"]
-| ((thermodynamic temperature)) | stem:[ii(T)] | stem:[Theta]
-| amount of substance | stem:[n] | stem:[sf "N"]
-| luminous intensity | stem:[ii(I)_("v")] | stem:[sf "J"] (((luminous intensity)))
+| time | stem:[t] | stem:[sf "&#x1D5B3;"]
+| ((length)) | stem:[l, x, r], etc. | stem:[sf "&#x1D5AB;"]
+| ((mass)) | stem:[m] | stem:[sf "&#x1D5AC;"]
+| ((electric current)) | stem:[ii(I), i] | stem:[sf "&#x1D5A8;"]
+| ((thermodynamic temperature)) | stem:[ii(T)] | stem:[sf "&#x03F4;"]
+| amount of substance | stem:[n] | stem:[sf "&#x1D5AD;"]
+| luminous intensity | stem:[ii(I)_("v")] | stem:[sf "&#x1D5A9;"] (((luminous intensity)))
```

Comment:
Question for @Intelligent2013 : do we even need `stem:[sf ...]` here? Or just `stem:[...]` would work? Ping @opoudjis too.

Reply:
@ronaldtse yes, you're right. It can be replaced by
```adoc
stem:["&#x1D5B3;"]
```
resulted xml:
```xml
<mtext>&#120243;</mtext>
```
 or 
```adoc
stem:[&#x1D5B3;]
```
resulted xml:
```xml
<mi>&#120243;</mi>
```
Both cases render identically:
![image](https://user-images.githubusercontent.com/5559926/125764326-7f02a6ba-4e64-4ee1-9ad2-fc886098cf89.png)

But first case more preferrable, because regarding MathML rules, single `mi` should be displayed as italic, but really it should be displayed as normal.

Reply:
That's great, thanks @Intelligent2013 !